From 0c95d6cedfdc4bccbad0fcd1309ce630ebdb04be Mon Sep 17 00:00:00 2001
From: Luca Magrone <luca@magrone.cc>
Date: Tue, 18 Oct 2022 00:41:11 +0200
Subject: [PATCH] Build with rpm CFLAGS

The makefile does not use a standard configure script so the CFLAGS variable
must be added to OPTIONS_COMPILE_DEBUG options list

Signed-off-by: Luca Magrone <luca@magrone.cc>
---
 src/makefiles/linux_64bit.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefiles/linux_64bit.mak b/src/makefiles/linux_64bit.mak
index 02f67e98..218d3833 100644
--- a/src/makefiles/linux_64bit.mak
+++ b/src/makefiles/linux_64bit.mak
@@ -62,7 +62,7 @@
 
 #CC=gcc
 
-OPTIONS_COMPILE_DEBUG=-D_DEBUG -DDEBUG -DUNIX -DUNIX_LINUX -DCPU_64 -D_REENTRANT -DREENTRANT -D_THREAD_SAFE -D_THREADSAFE -DTHREAD_SAFE -DTHREADSAFE -D_FILE_OFFSET_BITS=64 -I./src/ -I./src/Cedar/ -I./src/Mayaqua/ -g -fsigned-char -m64
+OPTIONS_COMPILE_DEBUG=${CFLAGS} -D_DEBUG -DDEBUG -DUNIX -DUNIX_LINUX -DCPU_64 -D_REENTRANT -DREENTRANT -D_THREAD_SAFE -D_THREADSAFE -DTHREAD_SAFE -DTHREADSAFE -D_FILE_OFFSET_BITS=64 -I./src/ -I./src/Cedar/ -I./src/Mayaqua/ -g -fsigned-char -m64
 
 OPTIONS_LINK_DEBUG=-g -fsigned-char -m64 -lm -ldl -lrt -lpthread -lssl -lcrypto -lreadline -lncurses -lz
 
-- 
2.37.3

